sql/driver/conn.go

```diff
@@ -55,6 +56,7 @@ func (c *conn) Prepare(query string) (driver.Stmt, error) {
 }
 
 func (c *conn) Begin() (driver.Tx, error) {
+	c.session = []byte{}
```

Comment:
You clear the session every time a transaction is started? That doesn't seem right. Btw, better to clear the session using `c.session = nil`.